src/vmware/azext_vmware/custom.py

```diff
@@ -6,6 +6,20 @@
 from knack.util import CLIError
 from azext_vmware.vendored_sdks.avs_client import AVSClient
 
+LEGAL_TERMS = '''
+LEGAL TERMS
+
+Azure VMware Solution is an Azure Service licensed to you as part of your Azure subscription and subject to the terms and conditions of the agreement under which you obtained your Azure subscription (https://azure.microsoft.com/support/legal/). The following additional terms also apply to your use of AVS:

```

Comment:
hi @ctaggart , based on one email, someone from CELA suggest below 
```
Azure VMware Solution (AVS) is an Azure Service licensed to you as part of your Azure 
```

Could you pls help to confirm? thanks

Reply:
Confirmed & fixed.